version.hpp

```diff
@@ -0,0 +1,23 @@
+// ======================================================================
+// \title  version.hpp
+// \author mstarch
+// \brief  fprime version information. MUST BE UPDATED for each release.
+//
+// \copyright
+// Copyright 2009-2015, by the California Institute of Technology.
```

Comment:
If this is a new file, is this block's date really correct?

Reply:
Old template. I will fix.